pyproject.toml

```diff
@@ -44,6 +44,7 @@ exclude = [
     "__pycache__",
     "benchmark_internal/others",
     ".venv",
+    "examples",
```

Comment:
why do you want this?

Reply:
@shivam15s removed.

I'll open an PR against https://github.com/linkedin/Liger-Kernel/pull/539#issuecomment-2673775793.
<img width="894" alt="Screenshot 2025-02-21 at 3 39 55 PM" src="https://github.com/user-attachments/assets/b0253135-257b-427e-8b04-939f840bbb5f" />